kak-lsp.toml

```diff
@@ -1,4 +1,4 @@
-snippet_support = false
+snippet_support = true
```

Comment:
?

Reply:
I accidentally copied my config, going to revert that line with the squashed commits later